geometry/geometry_config.py

```diff
@@ -389,7 +389,7 @@
               c.muShield.dZ5 + c.muShield.dZ6 +
               c.muShield.dZ7 + c.muShield.dZ8
         ) + c.muShield.LE
-        c.muShield.z = -(c.decayVolume.length + c.muShield.length) / 2.
+        c.muShield.z = -(c.decayVolume.length + c.muShield.length) / 2. + 1.6*u.m
```

Comment:
Why is this change needed?

Reply:
This is to move MuonShield closer to the DecayVolume (just for a time being) . The space provided for the TauNu detector was larger than what we anticipate now (5 m). This will probably change with a combi MuonShield.

Reply:
Ah, I see.